docs/indexing_integration_test.md

```diff
@@ -13,8 +13,10 @@ There's a script in `scripts/indexing_integration_test/manage_things_with_ids.py
 ```
 python manage_things_with_id.py -d "db_url" dump --input_file /tmp/source_ids --output_file /tmp/dumped_things
```

Comment:
I am assuming that this db_url here would be the database in the "local docker isB container", is this right?

Reply:
yeah, though I've actually been using the config option when I've been testing:

e.g.

```
python scripts/indexing_integration_test/manage_things_with_ids.py --config ./isb.cfg load --input_file /tmp/dumped_things.csv
```